x-pack/test/functional/apps/infra/home_page.ts

```diff
@@ -426,8 +426,8 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
           expect(nodesWithValue).to.eql([
             { name: 'host-5', value: 10, color: '#6092c0' },
             { name: 'host-4', value: 30, color: '#9ab6d5' },
-            { name: 'host-1', value: 50, color: '#f1d9b9' },
-            { name: 'host-2', value: 70, color: '#eba47a' },
+            { name: 'host-1', value: 50, color: '#f6e0b9' },
```

Comment:
These values change because we updated how colors are calculated. The changes should be pretty much indistinguishable.

* `#f1d9b9` -> `#f6e0b9` 
* `#eba47a` -> `#eda77a`

Reply:
Thanks for doing this, was actually on my list of things we needed to look into with the change of the theme